pmm-app/src/pmm-dbaas/DBaaS.tsx

```diff
@@ -23,6 +46,7 @@ export const DBaaSPanel: FC = () => {
             key={index}
             label={tab.label}
             active={tab.key === activeTab}
+            style={tab.disabled ? styles.disabled : undefined}
```

Comment:
Just as reminder, after addressing [this](https://github.com/percona/grafana-dashboards/pull/981/files#r497362496) you should use `className` instead of `style` here?

Reply:
Tab component from Grafana ignores className (PR opportunity 👀)